File: programme/square-root-of-number/square-root-of-number.go
package main

import (
	"fmt"
	"math"
)



func main() {

  var input float64
  fmt.Println("Enter a float value : 5.")
  fmt.Scanln(&input)
	result := math.Sqrt(input)
	fmt.Printf("\nResult 5: %.5f", result)
	
}

File: programme/square-root-of-number/square-root-of-number.cpp
//Programme to calculate square root of a number in C++

#include <stdio.h>      /* printf */
#include <math.h>       /* sqrt */

int main ()
{
  double param, result;
  param = 25.0;
  result = sqrt (param);
  printf ("sqrt(%f) = %f\n", param, result);
  return 0;
}
